examples/send-and-receive-emails/send_and_receive_emails.bal

```diff
@@ -42,21 +42,21 @@ public function main() {
     }
 
     // Create the client with the connection parameters, host, username, and
-    // password. An error is received in failure. Default port number `995` is
+    // password. An error is received in a failure. The default port number `995` is
     // used over SSL with these configurations.
     email:PopClient|email:Error popClient = new ("pop.email.com",
         "reader@email.com", "pass456");
 
     if (popClient is email:PopClient) {
-        // Read the first unseen email received by the POP3 server. Nil is
-        // returned when there are no new unseen emails. In error cases an
+        // Read the first unseen email received by the POP3 server. `()` is
+        // returned when there are no new unseen emails. In error cases, an
         // error is returned.
         email:Email|email:Error? emailResponse = popClient->read();
```

Comment:
Same as the earlier comment on error handling. Change all similar places.

Reply:
Created a new PR fixing this issue #843